test.js

```diff
@@ -29,7 +29,11 @@ promise.then(function(result) {
             })   
         }*/
         gitscrub.grabReadMeAtRepo(objArr[0], function(result){
-            var b64string = result['content']  
+            if(result['message'] == "Not Found") {
+                console.log("No readme found.");
+                return;
```

Comment:
npm says to avoid return statements as this breaks flow, we should try to return an error in the callback instead. This calls for refactoring on my part